src/components/AddBook.js

```diff
@@ -23,24 +24,26 @@ const AddBook = () => {
   };
 
   return (
-    <div>
-      <h2>ADD NEW BOOK</h2>
+    <div className={styles.formCont}>
+      <h2 className={styles.title}>ADD NEW BOOK</h2>
       <form>
         <input
           type="text"
           placeholder="Book title"
           value={title}
           onChange={(e) => setTitle(e.target.value)}
+          className={styles.bookTitle}
           required
         />
         <input
           type="text"
           placeholder="Category"
           value={category}
           onChange={(e) => setCategory(e.target.value)}
+          className={styles.category}
           required
         />
-        <button type="submit" onClick={(e) => submitBookToStore(e)}>Add Book</button>
+        <button className={styles.btn} type="submit" onClick={(e) => submitBookToStore(e)}>ADD BOOK</button>
       </form>
     </div>
   );
```

Comment:
- [ ] You can also check the design of the add book button comparing yours and the original design.

Original design

![design](https://user-images.githubusercontent.com/55635977/150569722-e6f6e68c-4283-4693-88c5-80cd61c2bcde.png)

Your design

![design2](https://user-images.githubusercontent.com/55635977/150569740-34601adf-7311-43e5-8373-4417f2e64bed.png)

Reply:
Thank you @Emmyn5600, I have made sure that the Add Book section is as wide as the rest.